doc/start/os-recommendations.rst

```diff
@@ -26,8 +26,10 @@ Linux Kernel
   for kernel version guidance.
 
   Older kernel client versions may not support your `CRUSH tunables`_ profile
-  or other newer features of the Ceph cluster, requiring the storage cluster
-  to be configured with those features disabled.
+  or other newer features of the Ceph cluster, requiring the storage cluster to
+  be configured with those features disabled. For RBD, a kernel of *at least*
+  version 5.4 is the minimum necessary for reasonable support for RBD image
```

Comment:
Just for my education: aren't "at least" and "the minimum" redundant here?

Reply:
@idryomov, Yeah, this is probably a little pleonastic. To my native-English-speaking, thinking-about-English-every-conscious-moment-for-four-decades ear, this doesn't sound absolutely grammatically wrong, but it is definitely a pleonasm. I'll remove the "at least". Good looking out.